lib/Remote/HttpCommandExecutor.php

```diff
@@ -123,6 +123,7 @@ class HttpCommandExecutor implements WebDriverCommandExecutor
         ],
         DriverCommand::SUBMIT_ELEMENT => ['method' => 'POST', 'url' => '/session/:sessionId/element/:id/submit'],
         DriverCommand::SCREENSHOT => ['method' => 'GET', 'url' => '/session/:sessionId/screenshot'],
+        DriverCommand::ELEMENT_SCREENSHOT => ['method' => 'GET', 'url' => '/session/:sessionId/element/:id/screenshot'],
```

Comment:
This is a W3C only feature .

Reply:
But it can stay here, right? The `$w3cCompliantCommands` array below in the file (in current community branch) is used for overloading methods which have different params in W3C. Methods which are the same could stay here.

Reply:
I kept the existing code.

Reply:
But you're still referencing ELEMENT_SCREENSHOT when the constant has been renamed to TAKE_ELEMENT_SCREENSHOT.